code/game/objects/items/devices/radio/radio.dm

```diff
@@ -746,3 +754,102 @@ var/global/list/default_medbay_channels = list(
 /obj/item/device/radio/phone/medbay/New()
 	..()
 	internal_channels = default_medbay_channels.Copy()
+
+/obj/item/device/radio/random_radio
+	name = "Random wave radio"
+	desc = "Radio that can pick up message from secure channels."
```

Comment:
Please add long and proper description what oddities can be used there, how, and how it can improve radio.